spec/factories.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
```

Comment:
Definitely need you to show me how this works! I haven't used FB yet but it looks great.